api/proto/relay/relay.proto

```diff
@@ -70,7 +70,8 @@ message ChunkRequest {
 }
 
 // The reply to a GetChunks request.
-// Requests are fulfilled in all-or-nothing fashion. If any of the requested chunks are not found, the entire request will fail.
+// Requests are fulfilled in all-or-nothing fashion. If any of the requested chunks are not found, the entire request
```

Comment:
This probably better to be placed next to the request or the relevant field in the request message.

Reply:
moved